src/sentry/options/defaults.py

```diff
@@ -175,3 +175,6 @@
 
 # From 0.0 to 1.0: Randomly disable normalization code in interfaces when loading from db
 register('store.empty-interface-sample-rate', default=0.0)
+
+# Symbolicator
+register('symbolicator.minidump-refactor-projects-opt-in', type=Sequence, default=[])
```

Comment:
Can use a `sentry:` option and skip registering.

Reply:
I really want to avoid using this kind of option in the future (see knight trading incident)